examples/structured_data/advanced_techniques_for_structured_data.py

```diff
@@ -0,0 +1,472 @@
+"""
+Title: Advanced techniques for structured data
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/12/31
+Last modified: 2020/12/31
+Description: Exploring advanced modeling techniques for structured data.
+"""
+
+"""
+## Introduction
+
+This example demonstrates how to do structured data classification using advanced techniques like:
+1. [Wide & Deep](https://ai.googleblog.com/2016/06/wide-deep-learning-better-together-with.html) models
+2. [Deep & Cross](https://arxiv.org/abs/1708.05123) models
+
+The example covers using [tf.feature_column](https://www.tensorflow.org/api_docs/python/tf/feature_column)
```

Comment:
Suggestion:

- Use backticks for methods, functions, etc (code in text :https://developers.google.com/style/code-in-text?hl=en0)

```suggestion
The example covers using [`tf.feature_column`](https://www.tensorflow.org/api_docs/python/tf/feature_column)
```

I hope this renders well on keras.io @fchollet LMKWYT

Reply:
Update:

If your line 174 renders OK:

```
[`tf.data.Dataset`](https://www.tensorflow.org/guide/datasets) for training or evaluation.
```

then this suggestion for having consistent formatting should be fine too.